enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -18,127 +18,31 @@
     PolicyTypes
 )
 from enterprise_access.apps.subsidy_access_policy.models import SubsidyAccessPolicy
-from enterprise_access.apps.subsidy_request.models import (
-    CouponCodeRequest,
-    LicenseRequest,
-    SubsidyRequest,
-    SubsidyRequestCustomerConfiguration
-)
 
 logger = logging.getLogger(__name__)
 
 
-class SubsidyRequestSerializer(serializers.ModelSerializer):
+class SubsidyAccessPolicyResponseSerializer(serializers.ModelSerializer):
```

Comment:
jumped the gun a little - this isn't used yet, but will be in an ensuing branch.